packages/wallets/src/features/accounts/screens/ManualDocumentUpload/components/IdentityCardDocumentUpload/IdentityCardDocumentUpload.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Divider, Dropzone, WalletText, WalletTextField } from '../../../../../../components';
+import useDevice from '../../../../../../hooks/useDevice';
```

Comment:
I guess we do not have patterns yet to avoid that kind of chained ../../../../.. ?

Reply:
Nope 😓 , may cause conflicts with other packages if we use type alias now

Reply:
The same thoughts :) Maybe it's time to add some paths to tsconfig?

Reply:
Thinking of using different approach unlike other packages like this: https://medium.com/@leonardobrunolima/react-tips-working-with-relative-path-using-create-react-app-fe55c5f97a21